src/lib/build/Build.ts

```diff
@@ -147,10 +148,10 @@ export class Build {
           artifactData.commitMap[eventKey] = artifact.version
         })())
       }
-
       await Promise.all(promises)
-      if (await buildVersions.saveAsync()) {
-        await buildVersions.gitRepo.commitAndPushAsync('update build versions')
+      
+      if (await buildVersions.saveAsync(commitMessage)) {
+        await buildVersions.gitRepo.commitAndPushAsync(commitMessage)
```

Comment:
I think you may have discovered a redundant line of code here, since `buildVersions.saveAsync` already does call `this.gitRepo.commitAndPushAsync(message)`